package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "15.7.0",
+  "version": "15.7.2",
```

Comment:
I'd probably go for 15.8.0 here, but you don't have to increase the versions, we can release it in a separate step